Chandra/cmd_states/cmd_states.py

```diff
@@ -104,9 +105,22 @@ def add_trans(date=date, **kwargs):
         # if no key in kwargs is in the exclude set then update transition
         if not (exclude and set(exclude).intersection(kwargs)):
             transitions.setdefault(date, {}).update(kwargs)
+            transitions['last_date'] = date
     return add_trans
 
 
+def _make_pitch_cmds(datestart, datestop, sample_time=5000):
```

Comment:
Eh.  I think in this case the function provides a little useful context for the one-liner, so I'll leave it.